apps/src/applab/applab.js

```diff
@@ -1121,6 +1121,10 @@ Applab.execute = function() {
       var session = studioApp.editor.aceEditor.getSession();
       annotationList.attachToSession(session, studioApp.editor);
       annotationList.clearRuntimeAnnotations();
+      studioApp.editor.aceEditor.session.on("change", function () {
```

Comment:
Unfortunately, this is pretty difficult to accomplish. The reason being that as soon as we hit an error in AppLab we call onPuzzleComplete, which for our level tests is the point at which the test is done. For this test, what we really want is to run some validation after executing the code.

I spent a little time trying to hack together a way to do something like this. It's certainly possible, but there's no quick clean and easy way to accomplish it. As such, I'm punting on adding a test for the time being.